//trycatch : are used for Handling Error

let a=10;
let b=20;

console.log(a+b);
console.log(a*b);
console.log(a+b);
console.log(a+b);
try{
console.log(a+b);

}
catch(err){
console.log(err);


}
console.log(a+b);
console.log(a+b);
console.log(a+b);